### PR TITLE
Luan/yolo

### PR DESCRIPTION
**Pull request by luanlam97**

### CODE REVIEW

File: downstream_model/yolo/visualize.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+def visualize_yolo(image,detection_result) -> np.ndarray:
+    """Draws bounding boxes on the input image and return it.
+    Parameters
+    ----------
+      image: cv2 image 
+          The input RGB image.
+      detection_result: 
+          The list of all bounding boxes entities to be visualize from yolo.
+
+    Returns
```
Comment (PierceLovesee) [inline downstream_model/yolo/visualize.py:23]: add in the description and returned datatype for the ***bounding_box_data***

File: downstream_model/yolo/yolo.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            **parameter: dict of argument
+                An unpacked  dict of argument that are native from ultralics framework
+
+
+        Returns
+        ----------
+            A list of Result object
+        """
+
+        result = self.model.predict(data, **parameter)
```
Comment (PierceLovesee) [inline downstream_model/yolo/yolo.py:136]: Add in a comment here that says something like:

"# IMPORTANT FOR SOMETHING.  NOT SURE -- FIGURE IT OUT LATER, LOL" @JamesF2440

File: downstream_model/mediapipe/visulize.py [hunk 3]
```diff
@@ -19,6 +19,7 @@ def visualize(
   Returns:
     Image with bounding boxes.
```
[the remark this answers is not in the document]
Comment (luanlam97) [reply downstream_model/mediapipe/visulize.py:20]: resolved

File: downstream_model/mediapipe/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Yolo
+
+'You Only Look Once' a powerful computer vision model.
+
+### **Demo**
+
+1. Install the necessary requirements
+
+```console
+!pip install -r requirements_yolo.txt.txt
```
Comment (PierceLovesee) [inline downstream_model/mediapipe/README.md:10]: remove extra .txt

File: downstream_model/yolo/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# Yolo
+
+'You Only Look Once' a powerful computer vision model.
+
+### **Demo**
+
+1. Install the necessary requirements
+
+```console
+!pip install -r requirements_yolo.txt.txt
```
Comment (PierceLovesee) [inline downstream_model/yolo/README.md:10]: remove extra .txt

**Review by PierceLovesee — COMMENTED** [review]
Just two small typos on the READMEs to fix, then we can merge it into main.

**Review by PierceLovesee — APPROVED** [review]
Looks good! 👍

**Pull request merged**